6.3.модуль_math/6.3.5.py

```diff
@@ -0,0 +1,4 @@
+from math import *
```

Comment:
Импортировать целый модуль не лучшая практика.
```suggestion
from math import sin, cos, tan, radians
```